modules/ztp-worker-node-applying-du-profile.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// Epic CNF-5335 (4.11), Story TELCODOCS-643
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-additional-worker-apply-du-profile_{context}"]
+= Applying profiles to the worker node
+include::../_attributes/common-attributes.adoc[]
+
+You can configure the additional worker node with a DU profile.
+
+The expanded SNO cluster with a distributed unit (DU) profile is provisioned using the `PolicyGenTemplate` resources partitioned into common, group, and site-specific manifests. The Git repository for SNO that is linked to the ArgoCD `policies` application includes the following files:
```

Comment:
Please consider replacing "The Git repository" ->  "The ZTP Gitops"

Reply:
@nwaizer Changed to "GitOps ZTP pipeline" to stay consistent with the name in our published docs (e.g. [Installing the GitOps ZTP pipeline](https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp-deploying-disconnected.html#ztp-installing-the-gitops-ztp-pipeline_ztp-deploying-disconnected)